.github/workflows/_build-liminal-node.yml

```diff
@@ -0,0 +1,35 @@
+---
+#  This workflow builds aleph-node with feature liminal
+#  features enabled. Those binaries MUST NOT be used in the production.
+name: Build liminal node
+on:
+  workflow_call:
+
+jobs:
+  main:
+    name: Build liminal node
+    runs-on: self-hosted
+    env:
+      RUST_BACKTRACE: full
+      RUSTC_WRAPPER: sccache
+    steps:
+      - name: Checkout aleph-node source code
+        uses: actions/checkout@v3
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+
+      - name: Install WASM target
+        run: rustup target add wasm32-unknown-unknown
+
+      - name: Build test binary and runtime
+        run: |
+          cargo build --release -p aleph-node --features "liminal"
```

Comment:
should we use `--profile production` instead? if so, then the path in the next step should be adjusted to `target/production/aleph-node`

Reply:
The problem is that it makes longer the whole build by 8 minutes, and consumes our multi core self hosted runner in the most inefficient way, as LTO linking that lasts this 8 minutes occupies only one CPU. So we need to make sure this is necessaryto use production profile here

Reply:
for standard node ofc

Reply:
for standard node, it is already there https://github.com/Cardinal-Cryptography/aleph-node/blob/main/.github/workflows/_build-production-node-and-runtime.yml#L34